examples/provide-liquidity/components/wallet/Wallet.tsx

```diff
@@ -59,6 +67,26 @@ export const WalletSection = () => {
     openView();
   };
 
+  const getBalance = async () => {
+    const client = await getSigningStargateClient();
+    const balance = await client?.getBalance(address as string, 'OSMO');
```

Comment:
`OSMO`, is this part of the API to use symbols?

Reply:
ideally we can use `uosmo`, but maybe it's the API. I'm curious for sure ;)

Reply:
https://cosmos.github.io/cosmjs/latest/stargate/classes/StargateClient.html#getBalanceStaked

getBalance function use denom not symbol

Reply:
e, I tested it.

`import { asset_list, assets } from '@chain-registry/osmosis';
export const osmosisAssets: Asset[] = [
  ...assets.assets,
  ...asset_list.assets.filter((item) => item.display != 'usdc'),
].filter(({ type_asset }) => type_asset !== 'ics20');
export const osmosisAssetsList: AssetList[] = [
  {
    assets: osmosisAssets,
    chain_name: 'osmosis',
  },
];`

The previous developers had a reason to merge assets and asset_list. If you only use asset_list, some assets might not be found.

if  not merge

`import { asset_list, assets } from '@chain-registry/osmosis';
export const osmosisAssets: Asset[] =  assets.filter(({ type_asset }) => type_asset !== 'ics20');
const osmosisAssetsList = [asset_list];
`
![Uploading img_v3_02cf_80d029c8-0bc2-49f7-b160-886f3a5418hu.jpg…]()

Reply:
yes, denom is `uosmo`, not `OSMO` right?

Reply:
sorry, I made a mistake， it is uosmo,
<img width="627" alt="image" src="https://github.com/cosmology-tech/create-cosmos-app/assets/14268015/f60f0f2a-6230-4158-b34d-2d1133f26474">

Reply:
fix: https://github.com/stark-tech-space/create-cosmos-app/pull/3/commits